static/scss/landingpage4.scss

```diff
@@ -359,3 +359,8 @@ ul#as_seen_on {
         }
     }
 }
+
+#footer {

```

Comment:
We don't have an element with an id of `footer` anymore to my knowledge and a quick grep. Is this old code from moving something over?